gp-includes/formats/format-pomo.php

```diff
@@ -30,6 +30,13 @@ public function print_exported_file( $project, $locale, $translation_set, $entri
 		$filters['status'] = 'current';
 
 		foreach ( $entries as $entry ) {
+			// Convert NULL to empty string to prevent PHP 8 passing NULL errors.
+			$entry->translations = array_map(
+				function ( $translation ) {
+					return $translation ?: '';
```

Comment:
Please, remove the short ternary, because it is now allowed by PHPCS, and it throws an error.
```suggestion
					return $translation ? $translation : '';
```